aspnetcore/blazor/tooling.md

```diff
@@ -58,22 +55,32 @@ To create a Blazor app on Windows, use the following guidance:
 
 :::moniker range=">= aspnetcore-8.0"
 
-<!-- UPDATE 8.0 Cross-link SSR -->
-
 * For a Blazor Web App in the **Additional information** dialog:
 
-  * Interactivity with server rendering is enabled by default with the **Use interactive server components** checkbox.
-  * To enable interactivity with client rendering, select the **Use interactive WebAssembly components** checkbox.
-  * To include sample pages and a layout based on Bootstrap styling, select the **Include sample pages** checkbox. Disable this option for an empty project without Bootstrap styling.
+<!-- UPDATE 8.0 For RTM, the dropdown list label changes to 
+                **Interactive render mode** -->
 
-  If both the WebAssembly and Server render modes are selected, the template uses the Auto render mode. The Auto render mode initially uses the Server mode while the .NET app bundle and runtime are download to the browser. After the .NET WebAssembly runtime is activated, Auto switches to the WebAssembly render mode.
+  * **Interactivity type** dropdown list:
```

Comment:
Just a heads up that the label for this dropdown will change to "Interactive render mode" for .NET 8 GA.